data/processed/patents/total_inventions.csv

```diff
@@ -0,0 +1,201 @@
+,year,nuts_id,nuts_year_spec,total_inventions
+0,2013,UKC1,2013,157
```

Comment:
Have you run autonuts here? I'm not sure if `year` being same as `nuts_year_spec` would be correct

Reply:
Years now seem correct but we need to fix the first column